doc/nrf/shortcuts.txt

```diff
@@ -126,6 +126,8 @@
 
 .. |thingy53_sample_note| replace:: If you build this application for Thingy:53, it enables additional features. See :ref:`thingy53_app_guide` for details.
 
+.. |nrf5340_mesh_sample_note| replace:: For the nRF5340 and Thingy:53, the extended advertiser has to be set manually for the network core, because the Bluetooth® LE does not know that Bluetooth mesh is enabled when build for this core. This can be done by setting ``CONFIG_BT_EXT_ADV=y`` for the network core.
```

Comment:
```suggestion
.. |nrf5340_mesh_sample_note| replace:: For nRF5340 and Thingy:53, the extended advertiser has to be set manually for the network core, because the Bluetooth® Low Energy does not know that the Bluetooth mesh is enabled when built for this core. This can be done by setting ``CONFIG_BT_EXT_ADV=y`` for the network core.
```